packages/create-cloudflare/package.json

```diff
@@ -74,6 +74,7 @@
 		"execa": "^7.1.1",
 		"glob": "^10.3.3",
 		"haikunator": "^2.1.2",
+		"indent-string": "^5.0.0",
```

Comment:
I'm tempted too but I figured since they're small, using these gives us one less thing to worry about for now.

Reply:
that's fine by me, I honestly don't have a huge preference either way 🙂 :+1:

Reply:
Since this is a devDep rather than dep, it's less bad but looking at the package contents I have to roll my eyes just a little :-) — it's a single simple function.

On the other hand we need to be very careful about deps and consider them a liability. They don't add much and only poses a risk of supply chain attacks or other issues (remember leftpad?). 

In general the fewer deps we have, the lower the risk that something goes awry.